lisa/main.py

```diff
@@ -87,6 +87,14 @@ def initialize_runtime_folder(
     working_path: Optional[Path] = None,
     run_id: str = "",
 ) -> None:
+    global _runtime_root
+    
+    # Update _runtime_root if working_path is provided
```

Comment:
I don't understand why the log_path works, but working_path not. Can you explain it?